pandas/tests/copy_view/test_indexing.py

```diff
@@ -340,7 +340,7 @@ def test_subset_set_column_with_loc(using_copy_on_write, using_array_manager, dt
             # The (i)loc[:, col] inplace deprecation gets triggered here, ignore those
             # warnings and only assert the SettingWithCopyWarning
             with tm.assert_produces_warning(
-                SettingWithCopyWarning,
+                None,
```

Comment:
Should this be considered as a false negative? (I would expect a warning here, but in general we are not super consistent about when we warn and when not)

Reply:
i dont fully grok when we should be issuing the warning, but I think that no copies are being made here, so it would make sense to not get the warning